modules/parents/client/views/parent-feed.client.view.html

```diff
@@ -1,4 +1,14 @@
+<<<<<<< HEAD:modules/user-feeds/client/views/parent-feed.client.view.html
```

Comment:
You committed the merge confict!!!!!!!!!!!! Eternal shame for @shreyanirmalan :smiling_imp: